torch/nn/modules/linear.py

```diff
@@ -115,4 +116,109 @@ def __repr__(self):
             + ', in2_features=' + str(self.in2_features) \
             + ', out_features=' + str(self.out_features) + ')'
 
+
+class NoisyLinear(Module):
+    """Applies a noisy linear transformation to the incoming data.
+    During training:
+        :math:`y = (mu_w + sigma_w \cdot epsilon_w)x
```

Comment:
I believe this is inline math, if that's what you want. If you wanted it on a new line I **think** the syntax is 
```python
.. math:: <your math here>
```

But perhaps Soumith can confirm